### DISCUSSION TITLE
Why I was rejected

### DISCUSSION THREAD

**Discussion by seba9989** [community/community#75714] [GitHub Education]
### Hi everyone,

Hello, 
I would like to know why my submission to the GitHub Education Pack was rejected. I sent a photo of my school ID in the first submission and a screen shot of my electronic gradebook in the second version.
Thank you in advance for your help. 

P.S.  My school does not assign email addresses to students

**Comment by iambashir** [comment]
Respons #1
You can contact with GitHub support team.

**Comment by samus-aran** [comment]
Hi @seba9989  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
